packages/asc-ui/src/components/SearchBar/SearchBar.tsx

```diff
@@ -20,6 +20,7 @@ export interface SearchBarProps extends TextFieldProps, SearchBarStyleProps {
    * @param value
    */
   onSubmit?: (e: React.FormEvent) => void
+  autoFocus?: boolean
```

Comment:
Can you add this functionality to a story of `SearchBar.stories.tsx` as well?

Reply:
Done